communication-service/index.js

```diff
@@ -0,0 +1,98 @@
+require("dotenv").config();
+const express = require('express');
+const cors = require('cors');
+const socketio = require("socket.io");
+const app = express();
+const port = process.env.PORT || 8004
+const chatServer = require('http').createServer(app);
+const adminName = 'Admin';
+const STATUS_CODE_SUCCESS  = 200;
```

Comment:
if got time, can separate the constant values into another constant.js file then import the constants from there